CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 3.19)
+
+set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CXX_STANDARD_REQUIRED on)
+
+if (APPLE)
+    foreach (HOMEBREW_PKG openssl libressl sqlite)
+        execute_process(COMMAND brew --prefix ${HOMEBREW_PKG} OUTPUT_VARIABLE HOMEBREW_PREFIX OUTPUT_STRIP_TRAILING_WHITESPACE)
+        list(APPEND CMAKE_PREFIX_PATH "${HOMEBREW_PREFIX}")
+    endforeach ()
+endif ()
```

Comment:
This makes sure that libraries installed with [Homebrew](https://brew.sh/) are properly detected. No more user complains about libraries failing to get linked.